### ISSUE TITLE
[Adaptive Cards - iOS - MediaInColumnSet.json]: Screen Reader is not conveying the name for the button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8421]
### Target Platforms

iOS

### SDK Version

Adaptive Cards

### Application Name

1.0 (2.4.0-beta.23.1.8.1)

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.1" control and select it.
5. Navigate to 'Test' control and activate it.
6. Navigate to 'MediaInColumnSet.json' and activate it.
7. Card will appear.
8. Navigate to the 'Cat Image' button and observe the issue.

**Actual Result:**
While navigating the 'MediaInColumnSet.Json' screen, screen reader is not conveying the name for both buttons. It is announcing only as button for the two controls.

**Expected Result:**
Voice over should convey the descriptive name of both button and it should announce as **'Media player button** for first media player and **'Media Player which doesn't specify a poster button'**.

**Note:**
1. Same issue is repro under "ColumnSet.Input.ChoiceSet.VerticalStretch" screen.
2. Same issue repro throughout the application for the same type of control.

**User Experience:**
It will impact the screen reader user if the screen reader does not convey the name of the button as the user won't be able to know about the desired control.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/229737203-6d3ad775-435a-4ac2-97e6-2c72e6185970.mov

https://user-images.githubusercontent.com/95913748/228835117-23d25ced-531f-446f-8bc3-4295b4126f2f.MP4

### Card JSON

```JSON
NA
```
